packages/core/src/engine/EngineUtils.ts

```diff
@@ -229,7 +258,8 @@ const valueADNums = (v: Value<ad.Num>): ad.Num[] => {
       return [v.contents];
     }
     case "BoolV":
-    case "StrV": {
+    case "StrV":
+    case "ShapeListV": {
```

Comment:
Could you clarify why this is the empty list? Is the idea that, in `compileCompGraph` (the only place `valueADNums` is used), we're really just thinking of a flat list of shapes, instead of a tree of `Group`s? Perhaps you could add a code comment here explaining what's going on.

Reply:
Yup I can add a comment.